configure.ac

```diff
@@ -0,0 +1,66 @@
+AC_PREREQ([2.63])
+AC_INIT([cray-tests], [0.5])
+AC_CONFIG_AUX_DIR(config)
+AC_CONFIG_MACRO_DIR(config)
+AC_CONFIG_HEADERS(config.h)
+AM_INIT_AUTOMAKE([1.11 dist-bzip2 foreign -Wall -Werror subdir-objects])
+m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
+
+AC_CANONICAL_HOST
+
+case $host_os in
+*linux*)
+	;;
+*)
+	AC_MSG_ERROR([cray-tests only builds on Linux])
+	;;
+esac
+
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
+
+AM_PROG_LIBTOOL
+
+AC_PROG_CC
+AM_PROG_CC_C_O
+
+LT_INIT
+
+AC_ARG_WITH([libfabric],
```

Comment:
not now, but we should probably add code to use pkg-config m4 functions at some point for finding libfabric and pmi